inc/codeception/overrides.php

```diff
@@ -41,21 +41,26 @@ function tests_add_filter() {
  *
  * This filter adds a unique test uploads folder just for our tests to reduce load.
  */
-tests_add_filter( 'upload_dir', function( $dir ) {
+tests_add_filter( 'upload_dir', function ( $dir ) {
 	array_walk( $dir, function( &$item ) {
 		if ( is_string( $item ) ) {
 			$item = str_replace( '/uploads', '/test-uploads', $item );
 		}
 	} );
 	return $dir;
-} );
+}, 1000 );
 
 /**
  * Reindex ElasticPress on install.
  */
 tests_add_filter( 'plugins_loaded', function () {
 	global $table_prefix;
 
+	// Not required for acceptance or functional tests.
```

Comment:
Why is this not required for acceptance/functional tests ? I believe with this we'll be missing Elastic functionality during automated tests ?

Reply:
I'll check again but I was having some issues with it getting stuck in a loop of indexing, maybe a race condition additional requests were being triggered. I'd found that I could invoke `wp ...` etc commands directly in the tests using `exec` to handle it instead which was more robust.

It had the additional problem that if I still had a tab open in the admin it was making heartbeat requests etc and things got really weird with that `.test-running` file existing.

Reply:
We might be able to circumvent the need for this by relying on the install hook- say if the db populator command was actually `wp core multisite-install` 🤔

Reply:
Yeah I'm not having any luck avoiding the continuous loop when it's triggered via a web (or web-like) request. I think it wasn't an issue initially because we weren't clearing up the indexes afterwards or beforehand. Ideally this process shouldn't be triggered by a web request anyway, so it's not that we don't need it, but we should consider alternatives for acceptance and functional tests.